airbyte-db/lib/src/main/java/io/airbyte/db/DataTypeUtils.java

```diff
@@ -13,12 +13,18 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
 import java.util.function.Function;
 
 public class DataTypeUtils {
 
   public static final String DATE_FORMAT_PATTERN = "yyyy-MM-dd'T'HH:mm:ss'Z'";
   public static final DateFormat DATE_FORMAT = new SimpleDateFormat(DATE_FORMAT_PATTERN); // Quoted "Z" to indicate UTC, no timezone offset
+  public static final DateFormat DATE_FORMAT_UTC = new SimpleDateFormat(DATE_FORMAT_PATTERN);
+
+  static {
+    DATE_FORMAT_UTC.setTimeZone(TimeZone.getTimeZone("UTC"));
+  }
```

Comment:
@sherifnada, please, take a look for it. We figured out that cases with converting duration and milliseconds to date format depends on time zone, but in general we don't know time zone for incoming parameters, as far as described in debezium docs, it returns milliseconds in UTC time zone, so we update date type utils regarding it.

Reply:
@sashaNeshcheret Can you clarify what you mean? If we don't know the time zone for incoming records, why are we assuming they are UTC? Couldn't that lead to data corruption?

Reply:
@sherifnada We don't know time zone for incoming parameters in general cases, so we can not make any assumption and change existing DATE_FORMAT that used in a lot of places, so I've created DATE_FORMAT_UTC and set it to UTC time zone specially for debezium. In debezium docs I found that timestamps converted to milliseconds that are in UTC to avoid any affecting, so i assumed that all numbers in dates files are following this approach and expect number in UTC format.
If it is not right assumption and we still don't have info about time zone, we can convert it to String as is.

Reply:
@sashaNeshcheret ok so if we know 100% that the places where we use this format are going to be in UTC, then this is okay. My only concern is assuming timetsamps are UTC if they are not.

Reply:
I reverted changes with UTC date format, because in documentation we are not 100% sure that number and Duration can be in another format.